assets/stylesRoot/colors.ts

```diff
@@ -4,8 +4,10 @@ const colors = {
   primaryGray: 'rgba(0,0,0,0.1)',
   secondGray: 'rgba(0,0,0,0.2)',
   thirdGray: 'rgba(0,0,0,0.5)',
+  fourthGray: '#B7C0C6',
   warning: 'red',
-  green: 'green'
+  green: 'green',
+  primaryYellow: '#FBD24D',
```

Comment:
[suggestion] For normalizing purposes, it is better to maintain a single standard for colors. We generally prefer to use HEX, which you can also work with alpha channel for the opacity.